mitodl/ol-data-platform
# Pipeline for Open edX course structure documents

# What are the relevant tickets?
Partially addresses #856 

# Description (What does it do?)
This adds a pipeline definition for retrieving all of the course structure JSON documents from our API plugin and writes them out as JSON files into a date-stamped path in S3 for ingestion by Airbyte

# How can this be tested?
```
$ export DAGSTER_ENV=qa
$ dagster dev -d src/ol_orchestrate/ -f src/ol_orchestrate/definitions/edx/openedx_data_extract.py
```
That should launch the Dagster service locally which you can then interact with at http://localhost:3000
